build/lib/helm_push.sh

```diff
@@ -20,13 +20,12 @@ set -o pipefail
 
 IMAGE_REGISTRY="${1?First argument is image registry}"
 HELM_DESTINATION_REPOSITORY="${2?Second argument is helm repository}"
-IMAGE_TAG="${3?Third argument is image tag}"
+VERSION="${3?Third argument is the (semantic) version}"
```

Comment:
iirc this IMAGE_TAG was actually overridden by what's in GIT_TAG right? So we don't need to keep it and this change is fine.

Reply:
I don't think so, GIT_TAG was part of what went into IMAGE_TAG. But the only place it was used in this file, as in the generation of a filename (line 28).